.github/workflow_templates/create-market-place-pr/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.11
+
+RUN pip install "pydantic==1.5.1" "pygithub==1.58.0"
```

Comment:
Should this be via some sort of requierments that is updated by our dependentbot ? This can be bind mounted during creation of such pip install.

Reply:
I think yes, but I would create a separate repo just for it (example https://github.com/tiangolo/latest-changes), to be separate (we do not need `pygithub` for any development step, for example) 

I would put this as a future refactor task, do you agree?

Reply:
ok just add TODO somewhere so we don't forget.